test/wdio/README.md

```diff
@@ -27,7 +28,39 @@ This allows to make changes to the test and have it re-run automatically. If you
 
 All components have to be compiled into a lazy-loaded Stencil component before executing the WebdriverIO test. WebdriverIO runs a set-up script (see `test/wdio/setup.ts`) to register all compiled custom components and have them available in your test. No additional imports are required.
 
-## Writing Test
+## Creating a new Test Suite
+
+WebderiverIO tests live in subdirectories within `test/wdio` alongside the components that they test. This keeps everything colocated and tidy!
+
+To start a new test suite, first create a new, descriptively-named directory:
+
+```sh
+mkdir test/wdio/my-excellent-new-test-suite
+```
+
+These are descriptively named, with a title that is either a noun or verb that
+succintly describes the test (`svg-class`, `slot-nested-order`,
+`slot-hide-content`, etc).
+
+After you've got a directory to work with, create a new Stencil component in
+that directory which is a minimal example of the behavior you're trying to
+test. The convention is to write this to a file called `cmp.tsx` within your
+directory. You can set the tag name and component class name to whatever you
+like, but try to ensure that your naming is relevant to what is being tested
+(prefer `<slot-relocation>` over `<my-test-component-2>`).
+
+> [!IMPORTANT]
+> Your new component must have a unique tag name that does not collide with
+> another component's tag name!
+
+You'll want to also create a new test file in that same directory. Your file
+must end with `.test.tsx`. The convention is to take the filename used for the
+test suite's main component and change the extension, so if our component was
+saved in `cmp.tsx` we'd put our tests in `cmp.test.tsx`.
+
+Once you've created your component and test file proceed to write some tests!
```

Comment:
Non-blocking - It might be worth calling out folks should probably `npm run build` here